ios/Classes/SwiftAirshipPlugin.swift

```diff
@@ -452,7 +452,8 @@ public class SwiftAirshipPlugin: NSObject, FlutterPlugin {
             return try AirshipJSON.wrap(flag).unWrap()
 
         case "featureFlagManager#trackInteraction":
-            guard let arg = try? call.requireAnyArg() as? String else {
+            guard let argument = try? call.requireAnyArg(),
```

Comment:
we have a requireStringArg, maybe that would clena things up?

Reply:
I'll try that